tests/bluetooth/shell/testcase.yaml

```diff
@@ -289,14 +289,22 @@ tests:
     build_only: true
     platform_allow: native_posix
     extra_configs:
-      - CONFIG_BT_TBS_CLIENT=n
+      - CONFIG_BT_TBS_CLIENT_TBS=n
+      - CONFIG_BT_TBS_CLIENT_GTBS=n
+    tags: bluetooth
+  bluetooth.shell.audio.tbs_only_client:
+    extra_args: CONF_FILE="audio.conf"
+    build_only: true
+    platform_allow: native_posix
+    extra_configs:
+      - CONFIG_BT_TBS_CLIENT_GTBS=n
```

Comment:
This would not be a valid config as per the TBS spec

Reply:
It is not mandatory for the Call Control Client to support GTBS. For Call Control Server yes, but not for the client

Reply:
Hmm, you are correct, but that sounds like an oversight, right? Two CCP devices (one server and one client) may not be able to interact at all if the CCP Server only has the mandatory GTBS and the CCP Client only supports TBS. 

I think I'll raise an erratum for this, as I (strongly) assume that the CCP Client should also mandatory support GTBS for interoperability

Reply:
I think it's not an oversight as e.g. you may want to implement controller for Skype application only.
Think about TV remote controllers, there are Generic controllers you may buy that will work with most of the vendors, but each vendor gives you controller that is especially for this TV

Reply:
I've raised https://bluetooth.atlassian.net/browse/ES-23589